client/lib/create-cached-selector/test/index.js

```diff
@@ -0,0 +1,192 @@
+/** @format */
+import { filter } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createCachedSelector, { MixedMap } from '../';
+import warn from 'lib/warn';
+
+jest.mock( 'lib/warn', () => jest.fn() );
+
+describe( 'index', () => {
+	describe( '#createCachedSelector', () => {
```

Comment:
I think that the root of this file will already appear as `create-cached-selector` so this two-level descent may report in the tests as `create-cached-selector/index/#createCachedSelector`

Reply:
I created a couple of errors to show how it outputs:
<img width="728" alt="screen shot 2017-12-19 at 8 04 02 pm" src="https://user-images.githubusercontent.com/4656974/34186123-d960aadc-e4f7-11e7-8853-a4d7292439e3.png">


What do you think?